tests/testthat/test-lancelot-multistrain.R

```diff
@@ -3515,3 +3516,144 @@ test_that("rotate strain uses correct variables", {
     setdiff(rotate_strain_compartments, names(which(n_strain_dim)))
   expect_length(variables_rotated_but_no_strain, 0)
 })
+
+
+test_that("Transmission, vaccine efficacy and cross immunity have
```

Comment:
There is no vaccination in these tests so we should remove all reference to vaccine efficacy/VE